doc/enterprise-shield/c_enterprise_shield_use_cases.md

```diff
@@ -0,0 +1,78 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Security with KAAZING Gateway
+
+Common Enterprise Shield™ Use Cases ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+=====================================
+
+This topic provides a high-level overview of some common Enterprise Shield™ use cases.
```

Comment:
We don't need the little TM trademark symbol for every occurrence of "Enterprise Shield". Headings  and the first occurrence on each page are sufficient.